examples/misc/patheffect_demo.py

```diff
@@ -8,7 +8,7 @@
 import matplotlib.patheffects as PathEffects
 import numpy as np
 
-if 1:
+if __name__ == "__main__":
```

Comment:
There's no need for this `if`; there's nothing in the example without this code.